src/types.rs

```diff
@@ -10,6 +11,21 @@ use ethereum_consensus::{
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use std::{collections::HashMap, fmt};
 
+#[derive(Parser, Default, Debug)]
+#[clap(version, about = "Beacon API client")]
+pub struct CliArgs {
+    #[clap(short, long)]
+    pub node_url: String,
+    #[clap(short, long)]
+    pub endpoint: String,
```

Comment:
I'm not sure what these are `node_url` vs `endpoint`

you could add comments here, or perhaps even add some example commands to the `README` to make it clear what we mean

Reply:
ah ok, I just saw your first comment where you explain some of this

I'd suggest using "method" where you are using "endpoint", as "endpoint" to me is a common networking term that means approximately the same as "node url" (which is why I was confused in parent comment)